templates/default/openssh.conf.erb

```diff
@@ -108,4 +108,7 @@ PermitLocalCommand no
 Compression yes
 
 #EscapeChar ~
-#VisualHostKey yes
\ No newline at end of file
+#VisualHostKey yes
+
+# http://undeadly.org/cgi?action=article&sid=20160114142733
+UseRoaming no
```

Comment:
Sure, I can do that. From what I understand this feature was never finished on the server-side, and was considered experimental on the client side.